LoRaEngine/LoraKeysManagerFacade/SyncDevAddrCache.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Host;
+    using Microsoft.Extensions.Logging;
+
+    public class SyncDevAddrCache
+    {
+        private LoRaDevAddrCache loRaDevAddrCache;
+        private RegistryManager registryManager;
+
+        public SyncDevAddrCache(LoRaDevAddrCache loRaDevAddrCache, RegistryManager registryManager)
+        {
+            this.loRaDevAddrCache = loRaDevAddrCache;
+            this.registryManager = registryManager;
+        }
+
+        [FunctionName("SyncDevAddrCache")]
+        public async Task Run([TimerTrigger("0 */5 * * * *")]TimerInfo myTimer, ILogger log)
+        {
+            log.LogDebug($"SyncDevAddrCache function executed at: {DateTime.Now}");
```

Comment:
Wonder if we should include `myTimer` to the logs. It has [interesting information](https://docs.microsoft.com/azure/azure-functions/functions-bindings-timer?tabs=csharp#usage)

Reply:
Sounds like a good idea to have the timer info on the logs

Reply:
I added some information to the log, let me know if you think it's enough